pykern/pkasyncio.py

```diff
@@ -0,0 +1,45 @@
+"""Wrapper on asyncio and tornado
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern import pkconfig
+from pykern.pkdebug import pkdlog
+import asyncio
+import tornado.gen
+import tornado.httpserver
+import tornado.ioloop
+import tornado.web
+
+
+async def sleep(secs):
+    await asyncio.sleep(secs)
+
+
+def tornado_http_server(uris, cfg):
```

Comment:
df7e434 and 0318990

I think keeping the list of coroutines to run is a bit strange. It would be nice to just have the event loop keep track of them. But, I don't know how to submit them to the event loop while it isn't running (before calling start). I'd appreciate your feedback.

Reply:
To be more specific - I could use [asyncio.get_event_loop](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop) but that is being deprecated and won't work in future version (will be aliased to get_running_loop and the loop won't be running). In addition, Tornado has moved away from directly interacting with the loop. So, I think this might be the only option...